### PR TITLE
feat: disable gravity bridge messages

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

Updates Gravity Bridge dependencie. That update disables most of the Gravity Bridge messages.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,7 +46,17 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
 ## [Unreleased]
 
-## [v1.0.4](https://github.com/umee-network/umee/releases/tag/v1.0.4) - 2022-09-08
+## [v1.1.0](https://github.com/umee-network/umee/releases/tag/v1.1.0) - 2022-09-08
+
+### State Machine Breaking
+
+- Disable Gravity Bridge bridge messages.
+
+Improvements:
```
Comment (toteki) [inline CHANGELOG.md:55]:
```suggestion
### Improvements
```

**Review by toteki — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -46,7 +46,17 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
-## [v1.0.4](https://github.com/umee-network/umee/releases/tag/v1.0.4) - 2022-09-08
+## [v1.1.0](https://github.com/umee-network/umee/releases/tag/v1.1.0) - 2022-09-08
+
+### State Machine Breaking
+
+- Disable Gravity Bridge bridge messages.
```
Comment (adamewozniak) [inline CHANGELOG.md:53]:
```suggestion
- [1358](https://github.com/umee-network/umee/pull/1358/files) Disable Gravity Bridge bridge messages.
```

**Review by adamewozniak — APPROVED** [review]

**Comment by RafilxTenfen** [comment]
The error on e2e is expected 
`#1: failed to send ethereum claims to Cosmos chain: failed to CalculateGas: rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: paused due to Ethereum PoS migration and PoW fork: service is unavailable: invalid request`
the orchestrator is not healthy due to the PR itself, we disabled the GB transactions and peggo can not read it 

We could modify e2e tests to not run the orchestrator and also not send any tx between eth <-> umee

**Comment by robert-zaremba** [comment]
Let's leave it for now for v1. I will disable this test. 
@RafilxTenfen can you make a relevant change for v3 (`main`)? I will need to backport this changes to main later on anyway.

**Comment by RafilxTenfen** [comment]
Yes, I can collaborate in the branch where you backport this

**Pull request merged**